### PR TITLE
adding clarification for governance

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/open-cluster-management/backlog/issues/14087

### CODE REVIEW

File: governance/create_policy.adoc [hunk 1]
```diff
@@ -18,7 +18,7 @@ View the following sections:
 You can create a security policy from the command line interface (CLI) or from the console.
 Cluster administrator access is required.
 
-*Important*: You must define a placement policy and placement binding to apply your policy to a specific cluster. Enter a value for the _Cluster selector_ field to define a PlacementPolicy and PlacementBinding. View the definitions of the objects that are required for your {product-title-short} policy:
+*Important*: You must define a placement policy and placement binding to apply your policy to a specific cluster. Enter a valid value for the _Cluster selector_ field to define a `PlacementPolicy` and `PlacementBinding`. See https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#resources-that-support-set-based-requirements[Resources that support support set-based requirements] in the Kubernetes documentation for a valid expression. View the definitions of the objects that are required for your {product-title-short} policy:
```
Comment (gparvin) [inline governance/create_policy.adoc]: Change "placement policy" to "placement rule" and change `PlacementPolicy` to `PlacementRule`.

**Review by gparvin — COMMENTED** [review]
Found another reference to "placement policy".  See included comment

File: governance/create_policy.adoc (hunk 1 above)
Comment (dockerymick) [reply governance/create_policy.adoc]: @gparvin should it also be changed on line 24?

Comment (dockerymick) [reply governance/create_policy.adoc]: Great catch @gparvin I have updated the instance from placement policy to placement rule. Moving forward to merge

**Pull request merged**